src/src/components/Dashboard/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { commonText } from '../../localization/common';
+import { Widget } from '../Widget';
+
+export const Dashboard = (props: { closeHandler: () => void; widgets: Array<WidgetObj> }) => {
+    
+    // NOTE: We might need to change the spacing a bit to center header better
+    return (
+        <> 
+            <div className='flex flex-row' >
+                
+                <h1 className='mx-auto my-0 text-center'>DASHBOARD</h1>
```

Comment:
[non-blocking]

The `DASHBOARD` text is probably just temporary at this point, but for the, could you please add any text strings into the `common.tsx` file in this directory - https://github.com/maxxxxxdlp/calendar-stats/tree/main/src/src/localization

Then, you can call it in code like `commonText('yourKey')`

You can also provide optional attributes to that function (like `commonText('key', 1, 2, 3)`).

See the [`README.md`](https://github.com/maxxxxxdlp/calendar-stats/tree/main/src/src/localization#readme) for more documentation